### PR TITLE
Esbuild sass

### PR DESCRIPTION
**Pull request by lei0zhou**
this is an implementation for #25

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -24,7 +19,7 @@ document.addEventListener("DOMContentLoaded", TransientControl.setup);
 //
 import "phoenix_html";
 import { Socket } from "phoenix";
-import { LiveSocket, Browser } from "phoenix_live_view";
```
Comment (lei0zhou) [inline assets/js/app.js:27]: Browser is not exported in [phoenix_liveview js ](https://github.com/phoenixframework/phoenix_live_view/blob/f778e5bb1a4b0a29f8d688bbc6c0b7182dea51ca/priv/static/phoenix_live_view.esm.js#L4934)

File: config/config.exs [hunk 2]
```diff
@@ -35,6 +35,21 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+config :esbuild,
+  version: "0.23.1",
```
Comment (jc00ke) [inline config/config.exs:39]: I'm assuming this is the latest version?

File: config/config.exs [hunk 3]
```diff
@@ -35,6 +35,21 @@ config :logger, :console,
...
+  default: [
+    args: ~w(js/app.js --bundle --target=es2016 --outdir=../priv/static/assets),
+    cd: Path.expand("../assets", __DIR__),
+    env: %{"NODE_PATH" => Path.expand("../deps", __DIR__)}
+  ]
+
+config :dart_sass,
+  version: "1.77.8",
```
Comment (jc00ke) [inline config/config.exs:47]: I'm assuming this is the latest version?

File: mix.exs [hunk 4]
```diff
@@ -47,8 +47,10 @@ defmodule EpiViewpoint.MixProject do
       local_or_remote(:remote, :euclid, version: "~> 0.1", path: System.get_env("EUCLID_PATH", "../euclid")),
       {:bcrypt_elixir, "~> 2.1"},
       {:cowboy_telemetry, "~> 0.4"},
+      {:dart_sass, "~> 0.7", runtime: Mix.env() == :dev},
       {:ecto_sql, "~> 3.11"},
       {:eqrcode, "~> 0.1"},
+      {:esbuild, "~> 0.7", runtime: Mix.env() == :dev},
```
Comment (jc00ke) [inline mix.exs:53]: While `~> 0.7` will match `0.8` I'd prefer if we match on the latest minor version. Same for `:dart_sass` too.

**Review by jc00ke — CHANGES_REQUESTED** [review]

File: mix.exs (hunk 4 above)
Comment (lei0zhou) [reply mix.exs:53]: the latest `dart_sass` is 0.7.1, will update `esbuild` to 0.8

File: config/config.exs (hunk 3 above)
Comment (lei0zhou) [reply config/config.exs:47]: yep

File: config/config.exs (hunk 2 above)
Comment (lei0zhou) [reply config/config.exs:39]: yep

File: assets/js/app.js [hunk 5]
```diff
@@ -67,13 +62,6 @@ let liveSocket = new LiveSocket("/live", Socket, {
   params: { _csrf_token: csrfToken }
 });
 
-window.addEventListener("phx:page-loading-stop", (info) => {
```
[the remark this answers is not in the document]
Comment (lei0zhou) [reply assets/js/app.js:70]: going to `/example/page#my-section` will jump to that section

I think it is fixed [here ](https://github.com/phoenixframework/phoenix_live_view/blob/main/CHANGELOG.md#bug-fixes-48)

**Review by jc00ke — APPROVED** [review]

**Pull request merged**